WcaOnRails/vendor/assets/javascripts/moment-timezone-with-data.js

```diff
@@ -0,0 +1,1204 @@
+//! moment-timezone.js
```

Comment:
Why would we bring the whole file here? I see it's included in the assets pipeline, so one way of doing this would be installing it over npm with `yarn add moment-timezone` and requiring as follows:
```js
//= require moment-timezone/builds/moment-timezone-with-data
```
It should work because Rails looks into `node_modules` as well.

Reply:
Ha now realizing I missed this comment.
Wouldn't this bring a second copy of the `moment` package? (the one brought in by `momentjs-rails`, and the one brought in by yarn)

I've tried it and as far as I get the sprockets error, rails doesn't seem to look into `node_modules` :s